Sources/Tracking/CustomerIOImplementation+ScreenViews.swift

```diff
@@ -59,7 +59,7 @@ extension CustomerIO {
         }
 
         let addionalScreenViewData = CustomerIOImplementation.autoScreenViewBody?() ?? [:]
-        automaticScreenView(name: name, data: addionalScreenViewData)
+        screen(name: name, data: addionalScreenViewData)
```

Comment:
This change might break this feature? `automaticScreenView` has a different implementation then the `screen` function.

Reply:
@levibostian from the code it seems like it is calling the `screen` from implementation, can you point to the code chunk which it is pointing but has a different implementation than `screen`?

Reply:
```
 func automaticScreenView(
        name: String,
        data: [String: Any]
    ) {
        guard let logger = diGraph?.logger else {
            return
        }
        automaticScreenView(name: name, data: StringAnyEncodable(logger: logger, data))
    }

    // Designed to be called from swizzled methods for automatic screen tracking.
    // Because swizzled functions are not able to determine what siteId instance of
    // the SDK the app is using, we simply call `screen()` on all siteIds of the SDK
    // and if automatic screen view tracking is not setup for that siteId, the function
    // call to the instance will simply be ignored.
    func automaticScreenView<RequestBody: Encodable>(
        name: String,
        data: RequestBody
    ) {
        implementation?.screen(name: name, data: data)
    }
```

Reply:
^^^ Yes, this is the code I am referring to. 

My apologies. I read the comment above the `automaticScreenView` function but skipped to read the function body. I was not aware that the function body changed.